src/website/shared/github.py

```diff
@@ -20,37 +13,13 @@
 
 def get_gh(per_page: int = 30) -> Github:
     """
-    Initialize a GitHub API connection, using credentials when available.
+    Initialize a GitHub API connection
     """
 
-    credentials_dir = env.get("CREDENTIALS_DIRECTORY")
-
-    gh_auth: Auth.Auth | None = None
-
-    if credentials_dir is None:
-        logger.warning("No credentials directory available, using unauthenticated API.")
-    else:
-        logger.info(f"Using credentials directory: {credentials_dir}")
-
-        try:
-            with open(f"{credentials_dir}/GH_TOKEN", encoding="utf-8") as f:
-                gh_auth = Auth.Token(f.read().rstrip("\n"))
```

Comment:
@yannham https://github.com/Nix-Security-WG/nix-security-tracker/pull/566/files#r2123702381 this is a noop if `GH_APP_PRIVATE_KEY` is set, which it is in the deployment. So removing the personal access token makes no difference.